nasa-gibs/worldview
# WV-3399: Remove wrap PACE orbit and update outdated links

## Description

Fixes #WV-3399 

- Removed "wrapadjacentdays" for PACE Descending/Night orbit track
- Updated earthdata.nasa.gov links that were returning 404's after the move to the new web site design/architecture
- 
## How To Test

1. `git checkout wv-3399-remove-wrap-PACE-orbit`
2. `npm run build && npm start`
4. Add the PACE Descending/Night orbit track
5. Verify expected result - Descending/Night orbit track no longer appears across the dateline wings
6. Check updated links in relevant tour stories and relevant layer descriptions

@nasa-gibs/worldview
